src/nwb_conversion_tools/tools/neo/neo.py

```diff
@@ -0,0 +1,602 @@
+      - name – the name of this electrode
+      - device – the device that was used to record from this electrode
+      - description – Recording description, description of electrode (e.g., whole-cell, sharp, etc)
+      - comment: Free-form text (can be from Methods)
+      - slice – Information about slice used for recording.
+      - seal – Information about seal used for recording.
+      - location – Area, layer, comments on estimation, stereotaxis coordinates (if in vivo, etc).
+      - resistance – Electrode resistance COMMENT: unit: Ohm.
+      - filtering – Electrode specific filtering.
+      - initial_access_resistance – Initial access resistance.
+
+    Parameters
+    ----------
+    neo_reader ([type]): Neo reader
+    electrodes_ids (list): List of electrodes ids.
+    block (int, optional): Block id. Defaults to 0.
+
+    Returns
+    -------
+    list: List of dictionaries containing electrodes metadata.
+    """
+    return []
+
+
+def get_number_of_electrodes(neo_reader) -> int:
+    """Get number of electrodes from Neo reader."""
+    # TODO - take in account the case with multiple streams.
+    return len(neo_reader.header["signal_channels"])
+
+
+def get_number_of_segments(neo_reader, block: int = 0) -> int:
+    """Get number of segments from Neo reader."""
+    return neo_reader.header["nb_segment"][block]
+
+
+def get_command_traces(neo_reader, block: int = 0, segment: int = 0, cmd_channel: int = 0) -> Tuple[list, str, str]:
+    """
+    Get command traces (e.g. voltage clamp command traces).
+
+    Parameters
+    ----------
+    neo_reader : neo.io.baseio
+    block : int, optional.
+        Defaults to 0.
+    segment : int, optional
+        Defaults to 0.
+    cmd_channel : int, optional
+        ABF command channel (0 to 7). Defaults to 0.
+    """
+    try:
+        traces, titles, units = neo_reader.read_raw_protocol()
+        return traces[segment][cmd_channel], titles[segment][cmd_channel], units[segment][cmd_channel]
+    except Exception as e:
+        msg = ".\n\n WARNING - get_command_traces() only works for AxonIO interface."
+        e.args = (str(e) + msg,)
+        return e
+
+
+def get_conversion_from_unit(unit: str) -> float:
+    """
+    Get conversion (to Volt or Ampere) from unit in string format.
+
+    Parameters
+    ----------
+    unit (str): Unit as string. E.g. pA, mV, uV, etc...
+
+    Returns
+    -------
+    float: conversion to Ampere or Volt
+    """
+    if unit in ["pA", "pV"]:
+        conversion = 1e-12
+    elif unit in ["nA", "nV"]:
+        conversion = 1e-9
+    elif unit in ["uA", "uV"]:
+        conversion = 1e-6
+    elif unit in ["mA", "mV"]:
+        conversion = 1e-3
+    elif unit in ["A", "V"]:
+        conversion = 1.0
+    else:
+        conversion = 1.0
+        warnings.warn("No valid units found for traces in the current file. Gain is set to 1, but this might be wrong.")
+    return float(conversion)
```

Comment:
I had asked Luiz to this before, but this logic seems very generally applicable (not specific to neo). Could you make it a `util` function instead?

Reply:
we could but let's wait until we need it somewhere else